addons/mass_mailing_sms/views/mailing_mailing_views.xml

```diff
@@ -14,32 +14,30 @@
     </record>
 
     <record id="mailing_mailing_view_form_sms" model="ir.ui.view">
-        <field name="name">mailing.mailing.view.form.inherit.sms</field>
+        <field name="name">mailing.mailing.view.form.sms</field>
         <field name="model">mailing.mailing</field>
-        <field name="inherit_id" ref="mass_mailing.view_mail_mass_mailing_form"/>
         <field name="arch" type="xml">
-            <!-- Buttons / Actions -->
-            <xpath expr="//button[@name='action_launch']" position="attributes">
-                <attribute name="attrs">{'invisible': ['|', ('mailing_type', '!=', 'mail'), ('state', 'in', ('in_queue', 'sending', 'done'))]}</attribute>
-            </xpath>
-            <xpath expr="//button[@name='action_schedule']" position="before">
-                <button name="action_put_in_queue" type="object"
-                    string="Send" class="oe_highlight"
-                    attrs="{'invisible': ['|', '|', ('mailing_type', '=', 'mail'), ('state', 'in', ('in_queue', 'sending', 'done')), ('sms_force_send', '=', True)]}"
-                    confirm="This will send SMS to all recipients. Do you still want to proceed ?"/>
-                <button name="action_send_mail" type="object"
-                    string="Send Now" class="oe_highlight"
-                    attrs="{'invisible': ['|', '|', '|', ('mailing_type', '=', 'mail'), ('state', 'in', ('in_queue', 'done')), ('schedule_type', '=', 'scheduled'), ('sms_force_send', '!=', True)]}"
-                    confirm="This will send SMS to all recipients now. Do you still want to proceed ?"/>
-            </xpath>
-            <xpath expr="//div[@name='schedule_block']/div[hasclass('row')]" position="inside">
-                <div attrs="{'invisible': ['|', '|', ('mailing_type', '=', 'mail'), ('state', 'in', ['in_queue', 'sending', 'done']), ('schedule_type', '=', 'scheduled')]}">
-                    <label for="sms_force_send" string="Skip Queue"/>
-                    <field name="sms_force_send"/>
-                </div>
-            </xpath>
-            <!-- Headers / Warnings -->
-            <xpath expr="//header" position="after">
+            <form string="Mailing" class="o_mass_mailing_mailing_form">
+                <header style="min-height:31px;">
+                    <button name="action_launch" type="object" class="oe_highlight" string="Send"
+                        attrs="{'invisible': ['|', ('mailing_type', '!=', 'mail'), ('state', 'in', ('in_queue', 'sending', 'done'))]}"
+                        confirm="This will send the email to all recipients. Do you still want to proceed ?"/>
```

Comment:
Still talking about email, which is incorrect in SMS view. I see some other are updated, could you re-check once everything to ensure we don't talk about mail in any sms-based view ? Thanks !